lnrpc/routerrpc/router_server.go

```diff
@@ -706,7 +706,7 @@ func (s *Server) QueryProbability(ctx context.Context,
 	amt := lnwire.MilliSatoshi(req.AmtMsat)
 
 	mc := s.cfg.RouterBackend.MissionControl
-	prob := mc.GetProbability(fromNode, toNode, amt)
+	prob := mc.GetProbability(fromNode, toNode, amt, 0)
```

Comment:
Perhaps we should actually fetch the capacity information (or have the user supply it) so this is updated to give accurate information?

Reply:
Right, this is done in a later commit as you noticed